peer/brontide.go

```diff
@@ -2286,7 +2286,10 @@ func (p *Brontide) reenableActiveChannels() {
 	// channel is already active, the update won't be sent.
 	for _, chanPoint := range activePublicChans {
 		err := p.cfg.ChanStatusMgr.RequestEnable(chanPoint, false)
-		if err != nil {
+		if err == netann.ErrEnableManuallyDisabledChan {
+			peerLog.Infof("Channel(%v) was manually disabled, ignoring "+
```

Comment:
I think we should move this to the `Debugf` logging level. We've been working over time to make the default info logs more useful at a glance by cutting down on unnecessary logs at the level.